scripts/run-integration-tests.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# This script runs the integration tests. It starts the test environment, runs the tests,
+# saves the container logs to a file, and stops the containers.
+# Usage:
+#   ./script.sh            # Run the script with default options
+#   ./script.sh --build    # Re-build the docker images (-b or --build)
+
+log_file="/tmp/rafiki_integration_logs.txt"
+build_flag=""
+
+# Parse cli args
+while [[ $# -gt 0 ]]; do
+  case "$1" in
+    -b|--build)
+      build_flag="--build"
+      shift
+      ;;
+    *)
+      echo "Unknown option: $1"
+      exit 1
+      ;;
+  esac
+done
+
+pnpm --filter integration testenv:compose down --volumes
+pnpm --filter integration testenv:compose up -d --wait $build_flag
+pnpm --filter integration testenv:compose logs -f --tail=0 > "$log_file" 2>&1 &
+pnpm --filter integration test
```

Comment:
I think it would be nice to prevent `pnpm --filter integration test` from running at all if we had errors starting up in `pnpm --filter integration testenv:compose up -d --wait $build_flag`

Reply:
makes sense. added https://github.com/interledger/rafiki/pull/2380/commits/bb98b70f7402fee1d098d3cf9f0ae85cde5c0294